samples/highcharts/data-tools/googlesheets-html/demo.html

```diff
@@ -0,0 +1,25 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/data-tools.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<p>
+    Use the fields below to configure the GoogleSheetsConnector. It will load
```

Comment:
Is it ok to share this sheet so it's easily accessible and we see how it is structured?
https://docs.google.com/spreadsheets/d/1U17c4GljMWpgk1bcTvUzIuWT8vdOnlCBHTm5S8Jh8tw/edit#gid=0

Reply:
We can't do this, but it has a form, so you can test with your own Google Spreadsheets.